database/migrations/2024_09_06_074330_create_user_video_table.php

```diff
@@ -27,6 +27,6 @@ public function up(): void
      */
     public function down(): void
     {
-        Schema::dropIfExists('');
+        Schema::dropIfExists('user_video');
```

Comment:
whats this changes for?

Reply:
itu kayaknya habis nge pull dari dev sebelumnya mas...

Reply:
> itu kayaknya habis nge pull dari dev sebelumnya mas...

hapus aja, ngga perlu

Reply:
okey done mas